apps/swirl-docs/src/components/Tokens/ZIndexTokens.tsx

```diff
@@ -0,0 +1,15 @@
+import { ZIndexTokenCategories } from "@swirl/lib/tokens";
+import { getTokens } from "@swirl/lib/tokens/src/utils";
+import { FunctionComponent } from "react";
+import TokensList from "./TokensList";
+
+export const ZIndexTokens: FunctionComponent = () => {
+  const tokens = getTokens(
+    {
+      other: [],
```

Comment:
**question:** Is it safe to use "other" for zIndex. There might be other tokens inside the category.

Reply:
I used other here, because it is the type of Z-Index in our Tokens.

e.g.
```json
"z-0": {
    "value": "0",
    "type": "other",
    "filePath": "./tokens/tokens.light.json",
    "isSource": true,
    "original": {
      "value": "0",
      "type": "other"
    },
    "name": "z-0",
    "attributes": {},
    "path": [
      "z-0"
    ]
  },
```
Currently in the <code>"type": "other"</code> we only have zindex tokens.
But i agree with you that this is not a semantically nice solution. Should we rename this tokenpreview to "OtherTokens"?

Reply:
Maybe Lenni can give it another type in Figma.

Reply:
Alright Lenny will update the tokens!